editor/modes/visual-editor/index.js

```diff
@@ -40,6 +52,20 @@ class VisualEditor extends Component {
 		}
 	}
 
+	onKeyDown( event ) {
+		const { uids } = this.props;
+		const isEditable = [ 'textarea', 'input', 'select' ].indexOf( document.activeElement.tagName.toLowerCase() ) !== -1
```

Comment:
Starting to see these conditions pop up in a few PRs with minor variations. Seems prone to edge cases or errors. Tested utilities might be more appropriate.

Wondering if there's another way to test here:

- If there's no `activeElement` (or is the default, `document.body` I think)
- If there's no focused blocks